packages/core/src/Cardano/types/Script.ts

```diff
@@ -1,18 +1,26 @@
 /* eslint-disable no-use-before-define */
-
+import * as util from '../util';
 import { Ed25519KeyHash } from './Key';
 import { Slot } from '@cardano-ogmios/schema';
 
 /**
- * Native script type.
+ * Plutus script type.
  */
-export enum NativeScriptType {
-  RequireAllOf = 'all',
-  RequireAnyOf = 'any',
-  RequireMOf = 'atLeast',
-  RequireSignature = 'sig',
-  RequireTimeAfter = 'after',
-  RequireTimeBefore = 'before'
+export enum ScriptType {
+  Native = 'native',
+  Plutus = 'plutus'
+}
+
+/**
+ * native script kind.
+ */
+export enum NativeScriptKind {
+  RequireSignature = 0,
```

Comment:
Do these numbers correspond to something in cbor? If not I think strings is nicer

```suggestion
  RequireSignature = 'sig',
```

Reply:
These are the ids of the script kinds in CBOR, previously I had two enums, one with the numbers (needed to map from CBOR to core types) and one with the strings, but in the end I merged them in a single enum and used it in both places (mapping to and from CBOR)